pandas/core/groupby/groupby.py

```diff
@@ -671,6 +671,9 @@ def get_group(self, name, obj=None):
             the DataFrame to take the DataFrame out of.  If
             it is None, the object groupby was called on will
             be used
+        default : object
+            the default object in case the group requested is
+            not found
```

Comment:
Probably needs to end in a period.

Reply:
Should this be a more specific type? Would there ever be a reason to pass something else besides a `get_group` callable as the default?

Reply:
>Probably needs to end in a period.

Fixed, thanks!

Reply:
>Should this be a more specific type? Would there ever be a reason to pass something else besides a get_group callable as the default?

I think you could also return a fixed value, like `default=DataFrame([-1, 0])`?